lib/veri.ml

```diff
@@ -148,41 +149,49 @@ class ['a] t arch dis =
     method private update_event ev =
       SM.update (fun c -> c#register_event ev)
 
-    (** [resolve_var var] - returns a result, bound with [var].
-        Sequence of searches is the following:
-        1) among read events that occured at current step in the same context,
-           with the same variable;
-        2) among read events that occures at current step, in other context,
-           with the same variable;
-        3) in current context, for the same variable *)
-    method private resolve_var : var -> 'a r = fun var ->
+    (** [touch x] - return a value bound with [x] where
+        [x] is either address or variable. In each variant of x
+        an appropriative lookup order is applied.
+        An event could be emited. *)
+    method private touch x =
```

Comment:
bad naming - touch associates with a side effect, a touch may not return any values. Something like `find_value` should reflect the method purpose better.